src/js/game/hud/parts/screenshot_exporter.js

```diff
@@ -56,7 +56,17 @@ export class HUDScreenshotExporter extends BaseHUDPart {
         const dimensions = maxChunk.sub(minChunk);
         logger.log("Dimensions:", dimensions);
 
-        const chunkSizePixels = 128;
+        let chunkSizePixels = 128;
+
+        if (dimensions.x + dimensions.y > 256) {
+            if (dimensions.x > dimensions.y) {
+                chunkSizePixels = Math.floor(128 * (128 / dimensions.x));
```

Comment:
Why not:

`chunkSizePixels = Math.floor(128 * (128 / Math.max(dimensions.x, dimensions.y)));` 

Also I think it'd be simpler to do:

`const maxDimensions = Math.max(dimensions.x, dimensions.y)`;

Then:

`if (maxDimensions > 128)`

And then divide by `maxDimensions`

Reply:
yup, that makes a lot of sense